ctx_test.go

```diff
@@ -72,34 +76,32 @@ func TestContext_PathParam(t *testing.T) {
 
 	t.Run("reading non-int path param to int sends an error", func(t *testing.T) {
 		s := NewServer()
-		Get(s, "/foo/{id}", func(c ContextNoBody) (ans, error) {
-			id, err := c.PathParamIntErr("id")
-			if err != nil {
-				return ans{}, err
-			}
-			return ans{Ans: fmt.Sprintf("%d", id)}, nil
+		Get(s, "/foo/{id}", func(c ContextNoBody) (any, error) {
+			return c.PathParamIntErr("id")
 		})
 
 		r := httptest.NewRequest("GET", "/foo/abc", nil)
 		w := httptest.NewRecorder()
 
 		s.Mux.ServeHTTP(w, r)
 
-		require.Equal(t, 422, w.Code)
+		assert.Equal(t, http.StatusUnprocessableEntity, w.Code)
+		require.Equal(t, crlf(`{"title":"Unprocessable Entity","status":422,"detail":"path param id=abc is not of type int"}`), w.Body.String())
```

Comment:
Please consider using require.JSONEq

This way you don't have to use crlf I think